README.md

```diff
@@ -42,14 +44,11 @@ Create the configuration file for app runtime.
 
 Expose configuration file's path as an environment variable in `AUTH_APP_CONFIG_PATH`.
 
-***Important Note:*** The environment where the application will be deployed, the timezone MUST be as UTC. Also, make sure redis is version `6.*`
+***Important Note:*** The environment where the application will be deployed, the timezone MUST be as UTC. Also, make sure redis is version `7.*`
```

Comment:
> the timezone MUST be as UTC

If a server's /etc/timezone is 'Europe/Berlin' would this not work?

Reply:
It's an outdated information, thanks for catching this.